pkg/auth/authTokensGet.go

```diff
@@ -54,3 +65,35 @@ func getAuthTokensFromRestApi(apiClient *galasaapi.APIClient) ([]galasaapi.AuthT
 
 	return authTokens, err
 }
+
+func formatFetchedTokensAndWriteToConsole(authTokens []galasaapi.AuthToken, console spi.Console) error {
+
+	summaryFormatter := tokensformatter.NewTokenSummaryFormatter()
+
+	outputText, err := summaryFormatter.FormatTokens(authTokens)
+
+	if err == nil {
+		console.WriteString(outputText)
+	}
+
+	return err
+
+}
+
+func validateLoginIdFlag(loginId string) (string, error) {
+
+	var err error
+
+	loginId = strings.TrimSpace(loginId)
+	splits := strings.Split(loginId, " ")
+
+	if loginId == "" {
+		err = galasaErrors.NewGalasaError(galasaErrors.GALASA_ERROR_MISSING_USER_LOGIN_ID_FLAG)
+	}
+
+	if len(splits) > 1 {
```

Comment:
Check if the string contains a space instead of splitting? Would be more readable and won't need to use a slice